src/pages/EditRequestMerchantPage.js

```diff
@@ -33,11 +35,26 @@ function EditRequestMerchantPage({defaultMerchant, onSubmit}) {
         return errors;
     }, []);
 
+    useFocusEffect(
+        useCallback(() => {
+            focusTimeoutRef.current = setTimeout(() => {
+                if (merchantInputRef.current) {
+                    merchantInputRef.current.focus();
+                }
+                return () => {
+                    if (!focusTimeoutRef.current) {
+                        return;
+                    }
+                    clearTimeout(focusTimeoutRef.current);
+                };
+            }, CONST.ANIMATED_TRANSITION);
+        }, []),
+    );
+
```

Comment:
I was talking about keeping the clear timeout login either in function parameter of `setTimeout` or in useCallback return function. But, nvm similar use cases are made in multiple places in the app. So, going to proceed with my review.